load_kaldi_wavscp.m

```diff
@@ -19,13 +19,13 @@
     [uid,scp] = parse_wavscp(line);
     Scp(uid) = scp;
     %disp(j);
-    j = j + 1;
+    %j = j + 1;
     line = fgetl(istream);
 end
 fclose(istream);
 
 function [uid,scp] = parse_wavscp(line)
-    k = findstr(' ',line);
+    k = strfind(line,' ');
```

Comment:
MATLAB recommended using the unambiguous "strfind" instead of the ambiguous "findstr".